scripts/bump_deps.lua

```diff
@@ -178,19 +116,25 @@ local function write_cmakelists_line(symbol, kind, value)
   }, 'Failed to write ' .. deps_file)
 end
 
-local function explicit_create_branch(dep)
-  require_executable('git')
+local function ref(dependency_name, _ref)
+  -- Allow all variations of tree-sitter, treesitter, tree_sitter_c, tree_sitter-c etc
```

Comment:
nah, that's not necessary. all deps are lower-case, and if you are unsure, just use `--list`. This iS n_ot NI_M.

Reply:
I meant more variations between: `treesitter` vs `tree-sitter` as well as `treesitter-c` vs `treesitter_c`. If you still think it's overkill I'll remove it

Reply:
I still think it's overkill.